examples/vmi-with-generic-hook.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  annotations:
+    hooks.kubevirt.io/hookSidecars: '[{"args": ["--version", "v1alpha2"], "image":"registry:5000/kubevirt/sidecar-shim:devel",
```

Comment:
Is this the right image? From the documentation you are mentioning `example-hook-generic` not the `sidecar-shim`

Reply:
You're right. I started out with a "generic sidecar hook" as an end goal. But now it's more of a sidecar shim with config map. I'll rename things accordingly to make more sense.